third_party/nvfuser/csrc/executor_utils.cpp

```diff
@@ -1040,6 +1040,17 @@ std::tuple<NvrtcFunction, std::string, std::vector<char>> nvrtcCompile(
         "NVFuser Compile: arch check disabled, should not compile any kernel");
   }
 
+  auto external_code_path = std::getenv("PYTORCH_NVFUSER_EXTERNAL_SRC");
```

Comment:
I think it's better to do this replacement outside of this function so that this function stays to be a pure util function for compilation. It's used from `compileFusion`, and I think that's a better place to do this debug replacement.

Reply:
Right after here: https://github.com/csarofeen/pytorch/blob/devel/third_party/nvfuser/csrc/executor.cpp#L294